src/Symfony/Component/HttpClient/TraceableHttpClient.php

```diff
@@ -70,15 +70,9 @@ public function stream($responses, float $timeout = null): ResponseStreamInterfa
             throw new \TypeError(sprintf('%s() expects parameter 1 to be an iterable of TraceableResponse objects, "%s" given.', __METHOD__, get_debug_type($responses)));
         }
 
-        return $this->client->stream(\Closure::bind(static function () use ($responses) {
-            foreach ($responses as $k => $r) {
-                if (!$r instanceof TraceableResponse) {
-                    throw new \TypeError(sprintf('%s() expects parameter 1 to be an iterable of TraceableResponse objects, "%s" given.', __METHOD__, get_debug_type($r)));
-                }
-
-                yield $k => $r->response;
-            }
-        }, null, TraceableResponse::class), $timeout);
```

Comment:
isn't this just missing an `()` before the comma?

Reply:
Indeed ! I didn't think about this way 🙂